CMakeLists.txt

```diff
@@ -1,34 +1,63 @@
-cmake_minimum_required( VERSION 3.5.1 )
+cmake_minimum_required(VERSION 3.5.1)
 
-project(lab_02_blending)
+# Set project name.
+project(lab-image-blending)
```

Comment:
Jeg skulle til å si at det var veldig lite CMakesk å bruke bindestrek, men for the record ser det ut til at du har [ditt på det tørre](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#variables). (CMake ville vel uansett sagt fra personlig ganske tidlig hvis det var ulovlig.)

[Faktisk kan du bruke emojis i CMake-variable](https://izzys.casa/2019/02/everything-you-never-wanted-to-know-about-cmake/).